src/components/Header/_header.scss

```diff
@@ -100,10 +101,10 @@
   // Used for links that are directly in the menubar to span the full height
   height: 100%;
   // Text styles
-  font-size: rem(14px);
+  font-size: carbon--type-scale(1);
   font-weight: 400;
   letter-spacing: 0;
-  line-height: rem(18px);
+  line-height: carbon--type-scale(3);
```

Comment:
Instead of using the type scale to set individual values, we can delete all of these and instead use the "type style" via:

```diff
- font-size: x
- font-weight: x
- letter-spacing
+ @include carbon--type-style("body-short-01");
```

This will include all the relevant typography properties and conforms them to the carbon-prescribed values. [From the Carbon website](https://www.carbondesignsystem.com/guidelines/typography/productive):


> body-short-01
> Type: IBM Plex Sans
> Size: 14px / .875rem
> Line-height: 18px / 1.125rem
> Weight: 400 / Regular
> Letter-spacing: .16px